panoramapublic/src/org/labkey/panoramapublic/view/search/panoramaWebSearch.jsp

```diff
@@ -244,164 +286,205 @@
                 let organism = document.getElementById(organismItemId).value;
                 let instrument = document.getElementById(instrumentItemId).value;
 
+                let expSearchParams = "";
                 if (author) {
                     expAnnotationFilters.push(createFilter(authorsItemId, author));
-                    updateUrlFilters(null, authorsItemId, author);
+                    expSearchParams += "Targeted MS Experiment List." + "authors~containsoneof" + "=" + author + "&";
                 }
                 if (title) {
                     expAnnotationFilters.push(createFilter(titleItemId, title));
-                    updateUrlFilters(null, titleItemId, title);
+                    expSearchParams += "Targeted MS Experiment List." + "title~containsoneof" + "=" + title + "&";
                 }
                 if (organism) {
                     expAnnotationFilters.push(createFilter(organismItemId, organism));
-                    updateUrlFilters(null, organismItemId, organism);
+                    expSearchParams += "Targeted MS Experiment List." + "organism~containsoneof" + "=" + organism + "&";
                 }
                 if (instrument) {
                     expAnnotationFilters.push(createFilter(instrumentItemId, instrument));
-                    updateUrlFilters(null, instrumentItemId, instrument);
+                    expSearchParams += "Targeted MS Experiment List." + "instrument~containsoneof" + "=" + instrument;
+                }
+                if (expSearchParams !== "") {
+                    location.replace(window.location.href + "?" + expSearchParams);
```

Comment:
These values still need to be URI-encoded as they're being appended.

Reply:
Yes! I thought i had done that, must have reverted it somehow with my refactors
Now done, again